04/index.html

```diff
@@ -5,23 +5,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 <body>
-    
-    <nav>
-        <ul>
-            <li>
-                <a href="#">start</a>
+    <label for="toggle">
+    <div class="hamburger"></div>
+    </label>
+    <input type="checkbox" id="toggle"/>
+    <nav class="site-nav">
+        <ul class="site-nav__list">
+            <li class="site-nav__item">
+                <a class="site-nav__link" href="#">start</a>
             </li>
-            <li>
-                <a href="#">oferta</a>
-                <ul>
-                    <li><a href="">strony internetowe</a></li>
-                    <li><a href="">pozycjonowanie stron www</a></li>
+            <li class="site-nav__item site-nav__item--no-border">
+                <label class="label" for="offer">
+                    <a class="site-nav__link" href="#">oferta</a>
+                </label>
```

Comment:
Jesli klikasz w tekst to na mobile się nie poakzuje drugi poziom. Już dla zagnieżdzonego elementu użyłbym `:hover`, który na mobile jest interpretowany jako clikck.